exts/betterTooltip/betterTooltip.js

```diff
@@ -109,6 +269,17 @@
                 return _measuresLabelFormatFunction;
             };
 
+            /**
+             * Gets or sets the format string for the value of the measures. Used by the default label formater.
```

Comment:
Gets or sets the format string for the value of the measures. Used by the default label formatter.